package.json

```
@@ -44,5 +46,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "concurrently": "^6.4.0",
+    "json-server": "^0.17.0"
```

You should make all your packages fixed version, because in future your app maybe break if versions conflict.Pls read: https://the-guild.dev/blog/how-should-you-pin-dependencies-and-why
```suggestion
    "concurrently": "6.4.0",
    "json-server": "0.17.0"
```